### ISSUE TITLE
git submodule add give me a fatal error:

### ISSUE THREAD

**Issue by switzer** [kriswallsmith/FacebookBundle#1]
Hi - I am trying out your Facebook Bundle, and I am unable to add it to my symfony installation.  Here is the output from what I have done:

$ git submodule add git://github.com/kriswallsmith/FacebookBundle.git src/Bundle/Kris/FacebookBundle
Initialized empty Git repository in /Users/scott/code/magpi/src/Bundle/Kris/FacebookBundle/.git/
fatal: The remote end hung up unexpectedly
Clone of 'git://github.com/kriswallsmith/FacebookBundle.git' into submodule path 'src/Bundle/Kris/FacebookBundle' failed
$ 

I have Googled around for an answer, and most seem to be permissions errors.  I was able to add the Facebook PHP SDK without a problem.

Any ideas?  Much appreciated!

**Comment by kriswallsmith** [comment]
Try this…

```
git submodule add git://github.com/kriswallsmith/KrisFacebookBundle.git src/Bundle/Kris/FacebookBundle
```

I think the README is wrong.

**Comment by kriswallsmith** [comment]
Fixed README (closed by 6a364213e780fe614657364f22df5e20ecebf916)
